cfgov/apache/dockerfile

```
@@ -0,0 +1,24 @@
+FROM httpd:latest
```

Since we're only using Apache for parity with legacy, can we pin 2.4? And should we use Alpine?
```suggestion
FROM httpd:2.4-alpine
```